auth-web/src/components/auth/MaintainBusinessInfo.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <v-container id="maintain-info-container">
+    <v-row>
+      <!-- Info Column -->
+      <v-col cols="12" md="6">
+        <h2>Manage and Maintain Your Business</h2>
+        <v-list-item class="list-item" v-for="(item, index) in bulletPoints" :key="index">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Sub Bullet Points -->
+        <v-list-item class="list-item list-item-sub" v-for="(item, index) in subBulletPoints" :key="`Sub-index: ${index}`">
+          <v-icon size="6" class="list-item-bullet">mdi-square</v-icon>
+          <v-list-item-content>
+            <v-list-item-subtitle class="list-item-text">
+              {{item.text}}
+            </v-list-item-subtitle>
+          </v-list-item-content>
+        </v-list-item>
+        <!-- Panel Btns -->
+        <div class="maintain-info-btns">
+          <v-btn v-if="userProfile" large color="#fcba19" class="my-5"
```

Comment:
Can you use colour variable here?
`$BCgovGold5: #fcba19; // PRIMARY GOLD`

Reply:
Same thing here, assigning a class and colour through css doesn't seem to carry through into the component. For whatever reason the colour needs to be passed as a prop and it doesn't register our theme colour references...